memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
+
+## Quick Start
+
+This section describes how to run YCSB on memcached.
+
+### 1. Install and start memcached service on the host(s)
+
+### 2. Install Java and Maven
+
```

Comment:
That should be ok for now. I agree it should probably go into a single documented place. The top level README is an option.